lib/basedriver/driver.js

```diff
@@ -38,7 +38,7 @@ class BaseDriver extends Protocol {
    */
   static baseVersion = BASEDRIVER_VER;
 
-  constructor (opts = {}, shouldValidateCaps = true) {
+  constructor (opts = {}, driverArgs = {}, shouldValidateCaps = true) {
```

Comment:
this might break existing drivers. Consider adding a new argument to the end rather than in the middle